instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import CommentSection from '../CommentSection/CommentSection';
+import Post from '../Post/Post';
+import '../Post/Post.css'
+
+const PostContainer = props => {
+  return (
+    <div className="post-container">
+      {props.posts.map((post) => {
+        return(
+          <div>
+            <Post likes={post.likes} image={post.imageUrl} thumbnail={post.thumbnailUrl} username={post.username} key={post.timestamp} />
+            <CommentSection comments={post.comments} key={post.id} />
```

Comment:
You can format this differently to make it more readable like so:

`            <Post
              likes={post.likes}
              image={post.imageUrl}
              thumbnail={post.thumbnailUrl}
              username={post.username}
              key={post.timestamp}
            />

            <CommentSection
              comments={post.comments}
              key={post.id}
              />`